localhost.go

```diff
@@ -156,7 +156,12 @@ func (c *Localhost) command(cmd string, o *exec.Options) (*osexec.Cmd, error) {
 		return osexec.Command("cmd.exe", "/c", cmd), nil
 	}
 
-	return osexec.Command("bash", "-c", "--", cmd), nil
+	shell := os.Getenv("SHELL")
```

Comment:
I wonder how this affects people with something like zsh or fish as `SHELL` 🤔

Reply:
We could also just use "sh" directly then.

Reply:
sh is not always bourne compatible, there are many distros that have dash as /bin/sh

Reply:
That's the point. Trying to stick with POSIX is probably the most portable thing to do.

Reply:
Sounds logical. I wonder if there are many exec calls in k0sctl/launchpad that run some oneliners that won't work via `sh` 🤔

Reply:
Hardcoded `sh` for now.